db/install.xml

```diff
@@ -190,5 +190,23 @@
         <INDEX NAME="userid-forumid" UNIQUE="false" FIELDS="userid, forumid"/>
       </INDEXES>
     </TABLE>
+    <!-- table for forum actions -->
+    <!-- TODO: Will need to look at moving the table create here, rename table as well, will need to discuss this with team -->
+    <!-- <TABLE NAME="hsuforum_actions" COMMENT="Each record is one action (like or thanks) on a forum post">
+      <FIELDS>
+        <FIELD NAME="id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="true"/>
+        <FIELD NAME="postid" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
+        <FIELD NAME="userid" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
+        <FIELD NAME="action" TYPE="char" LENGTH="255" NOTNULL="true" SEQUENCE="false"/>
+        <FIELD NAME="created" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
+      </FIELDS>
+      <KEYS>
+        <KEY NAME="primary" TYPE="primary" FIELDS="id"/>
+      </KEYS>
+      <INDEXES>
+        <INDEX NAME="postid" UNIQUE="false" FIELDS="postid"/>
+      </INDEXES>
+    </TABLE> -->
+    <!-- table for forum actions -->
```

Comment:
Lets discuss as outlined. Feel we cannot alter this table since this will affect existing actions for prod.

Reply:
I wanted to added on install for new instances as we won't have hsuforum_actions install to run at a later stage. brought this up in the initial chat we had with BSA.

Reply:
Do not follow here. Lets discuss when you are available